src/warehouse/schemas.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2018, Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from marshmallow import Schema, fields
+
+
+class StrictSchema(Schema):
+    class Meta:
+        strict = True
+
+
+class Base(StrictSchema):
```

Comment:
I would append `Schema` to all of these class names in order to not confuse them with the database models.

Reply:
I do disagree here. I think it makes sense to have a single namespaced name, because within your namespace, you shouldn't have to care whether other modules might use the same name. Postfixing the type seems a bit redundant. I'd prefer to have, for example:

* `models.Experiment`
* `schemas.Experiment`
* `resources.Experiment`

Then, if one module imports a class from another, it can either namespace them by importing the module, e.g., `from . import schemas` (my preferred option), or alias the import, e.g., `from .schemas import Experiment as ExperimentSchema`.